zcash_keys/Cargo.toml

```
@@ -21,7 +21,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 [dependencies]
 zcash_address.workspace = true
 zcash_encoding.workspace = true
-zcash_primitives.workspace = true
+zcash_transparent.workspace = true
```

```suggestion
transparent.workspace = true
```
and move this down to the "Transparent protocols" section of dependencies.